extension/src/experiments/commands/index.ts

```diff
@@ -108,11 +108,11 @@ export const getShareExperimentAsCommitCommand =
   }
 
 export const getShareExperimentToStudioCommand =
-  (internalCommands: InternalCommands, connect: Connect) =>
+  (internalCommands: InternalCommands, setup: Setup) =>
```

Comment:
[F] All of the externals of `Connect` have been moved directly onto `Setup` to reduce the size of this change. Will work out a smarter split once the dust settles.